package.json

```diff
@@ -50,6 +50,8 @@
     "wsrun": "^5.2.4"
   },
   "dependencies": {
+    "@types/js-yaml": "^3.12.5",
+    "github-label-sync": "^2.0.2",
```

Comment:
quick note: should be part of devDependencies if at all, not in `dependencies` (as you can see we've been working pretty hard to make this list 'empty' here -- can elaborate on why that is in sync conv :)).

Reply:
Ok, no problem.  Updated.